pdfminer/utils.py

```diff
@@ -293,6 +293,21 @@ def matrix2str(m):
     return '[%.2f,%.2f,%.2f,%.2f, (%.2f,%.2f)]' % (a, b, c, d, e, f)
 
 
+
+def mean(it):
+    sum=0
+    count=0
+    for el in it:
+        sum+=el
+        count+=1
+    return sum / count
```

Comment:
You could compute this with a readable oneliner:

```python
return sum(it) / len(list(it))
```

Reply:
This function can work with finite length iterators. Though it turned out that isn't needed.